avocado/plugins/replay.py

```diff
@@ -88,9 +91,26 @@ def _valid_ignore(self, string):
         return ignore_list
 
     def load_config(self, resultsdir):
-        config = os.path.join(resultsdir, 'replay', 'config')
-        with open(config, 'r') as f:
-            settings.process_config_path(f.read())
+        config = jobdata.retrieve_config(resultsdir)
+        if config is not None:
+            settings.process_config_path(config)
+
+    def _create_replay_map(self, resultsdir, replay_filter):
```

Comment:
This could use some docstring. First I thought it should be part of the `jobdata`, but after studying the purpose a bit more it really belongs only here.